src/ng/compile.js

```diff
@@ -186,6 +186,27 @@
  *   you want to shallow watch for changes (i.e. $watchCollection instead of $watch) you can use
  *   `=*` or `=*attr` (`=*?` or `=*?attr` if the property is optional).
  *
+ * * `<` or `<attr` - set up one-way (one-directional) binding between a local scope property and the
+ *   parent scope property of name defined via the value of the `attr` attribute. If no `attr`
+ *   name is specified then the attribute name is assumed to be the same as the local name.
+ *   Given `<dir my-attr="parentModel">` and directive definition of
+ *   `scope: { localModel:'<myAttr' }`, then isolate scope property `localModel` will reflect the
+ *   value of `parentModel` on the parent scope. Any changes to `parentModel` will be reflected
+ *   in `localModel`, but changes in `localModel` will not reflect in `parentModel`. There are however
+ *   two caveats:
+ *     1. one-way binding does not copy the value from the parent to the isolate scope, it simply
+ *     sets the same value. That means if your bound value is an object, changes to its properties
+ *     in the isolate scope will be reflected in the parent scope.
+ *     2. one-way binding watches changes to the **identity** of the parent value. That is important should
+ *     you replace a bound object in the isolated scope. In that case, you must also replace your
+ *     object in ther parent scope with a new one, otherwise the one-way binding won't pick up changes.
+ *
+ *   One-way binding is useful if you do not plan to propagate changes to your isolated scope bindings
+ *   back to the parent. It does not however not make this completely impossible.
```

Comment:
Hehe, yeah the explanation was problematic to write. Basically, if you replace the bound object in the isolate scope, changing only a property of the object in the parent scope won't update the child scope, because the watch on the parent scope only picks up when the identity of the parent scope value changes.